airgun/views/repository.py

```diff
@@ -22,7 +26,7 @@
 )
 
 
-class RepositoriesView(BaseLoggedInView, SearchableViewMixin):
+class RepositoriesView(BaseLoggedInView, SearchableViewMixinPF4):
```

Comment:
@pondrejk Is foreman 3.9 (katello 4.11.1) (6.15.z) really using this? For me this breaks some of the following tests:

```
tests/foreman/ui/test_repository.py
```